com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,129 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=home
```

Comment:
why do you need `home`?

Reply:
I found the docs very confusing.  I still don't know what I'm doing.

Reply:
I removed the home and it seems to work.  It's saving data in ~/.var/  I assumed I needed home for that.